connectors/src/lib/models/zendesk.ts

```diff
@@ -445,21 +441,6 @@ ZendeskTicket.init(
       allowNull: false,
       validate: { throwOnUnsafeInteger },
     },
-    groupId: {
-      type: DataTypes.BIGINT,
-      allowNull: true,
-      validate: { throwOnUnsafeInteger },
-    },
-    assigneeId: {
-      type: DataTypes.BIGINT,
-      allowNull: true,
-      validate: { throwOnUnsafeInteger },
-    },
-    organizationId: {
```

Comment:
should this be cleaned? https://github.com/dust-tt/dust/blob/main/connectors/src/connectors/zendesk/lib/sync_ticket.ts#L96

Reply:
yes absolutely, pushing